cmd/kops/create_cluster.go

```diff
@@ -59,6 +59,8 @@ type CreateClusterOptions struct {
 	MasterSize           string
 	MasterCount          int32
 	NodeCount            int32
+	MasterVolumeSize     int32
```

Comment:
I understand, we need to document size of the int then. Meaning are we defining mb or gb in a int. We can do a follow-up pr.

You backed out the base volume changes, which we still need. Those changes should be outside of this PR. If I can recruit you for those that would be amazing, but no worries and I know you are busy.